docs/Cli.md

```diff
@@ -341,6 +341,8 @@ Re-apply any new defaults as desired.
 |  yaw_deadband  | 5 | These are values (in us) by how much RC input can be different before it's considered valid. For transmitters with jitter on outputs, this value can be increased. Defaults are zero, but can be increased up to 10 or so if rc inputs twitch while idle. |
 |  throttle_tilt_comp_str  | 0 | Can be used in ANGLE and HORIZON mode and will automatically boost throttle when banking. Setting is in percentage, 0=disabled. |
 |  flaperon_throw_offset  | 200 | Defines throw range in us for both ailerons that will be passed to servo mixer via input source 14 (`FEATURE FLAPS`) when FLAPERON mode is activated. |
+|  gimbal_mode  | NORMAL | When feature SERVO_TILT is enabled, this can be either NORMAL or MIXTILT |
+>>>>>>> fc9ab94... add CLRACINGF4AIRV3 TARGET
```

Comment:
There are conflict markers merged. Also, it looks like this touches a lot of places that it shouldn't.

Reply:
Hmm. Look like the rebase to development I messed up.  Will update later. Thanks.